google/cloud/bigquery/opentelemetry_tracing.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Google LLC All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from contextlib import contextmanager
+from google.api_core.exceptions import GoogleAPICallError
+
+Logger = logging.getLogger(__name__)
+try:
+    from opentelemetry import trace
+    from opentelemetry.trace.status import Status
+    from opentelemetry.instrumentation.utils import http_status_to_canonical_code
+
+    HAS_OPENTELEMETRY = True
+
+except ImportError:
+    Logger.info(
+        "This service instrumented using opentelemetry."
+        "Opentelemetry could not be imported please"
+        "add opentelemetry-api and opentelemetry-instrumentation"
+        "packages in order to get Big Query Tracing data"
+    )
+
+    HAS_OPENTELEMETRY = False
+
+
+class SpanCreator:
+    def __init__(self):
+        # Constructs a span creator with all default attributes
+        self.opentelemetry_enbled = HAS_OPENTELEMETRY
```

Comment:
Since these are both global properties, maybe an object isn't even required? Could `create` be made into just a module-level method? https://www.youtube.com/watch?v=o9pEzgHorH0

Reply:
yeah I think a module level method might make more sense will do